src/hue/event_data.rs

```diff
@@ -38,6 +38,28 @@ struct MotionUpdateData {
     motion: MotionData,
 }
 
+#[derive(Deserialize, Debug, Clone)]
+struct TemperatureData {
+    temperature: f64,
+}
+
+#[derive(Deserialize, Debug, Clone)]
+struct TemperatureUpdateData {
+    id: String,
+    temperature: TemperatureData,
+}
+
+#[derive(Deserialize, Debug, Clone)]
+struct LightLevelData {
+    light_level: f64,
+}
+
+#[derive(Deserialize, Debug, Clone)]
+struct LightLevelUpdateData {
+    id: String,
+    light_level: LightLevelData,
```

Comment:
I believe this field should be `light` instead of `light_level` based on the CLIP API debugger output you provided :-)